lib/spack/spack/solver/asp.py

```diff
@@ -535,10 +535,9 @@ def solve(
 
         # Initialize the control object for the solver
         self.control = clingo.Control()
-        self.control.configuration.solve.models = nmodels
-        self.control.configuration.asp.trans_ext = 'all'
-        self.control.configuration.asp.eq = '5'
```

Comment:
Any reason we can't still use these? I didn't think they conflicted with heuristics.